nab/detectors/htmjava/src/main/java/nab/detectors/htmjava/HTMModel.java

```diff
@@ -194,9 +204,12 @@ public Parameters getTemporalMemoryParams(JsonNode modelParams) {
         if (tpParams.has("predictedSegmentDecrement")) {
             p.set(KEY.PREDICTED_SEGMENT_DECREMENT, tpParams.get("predictedSegmentDecrement").asDouble());
         }
-        if (tpParams.has("seed")) {
-            p.set(KEY.SEED, tpParams.get("seed").asInt());
-        }
+        // if (tpParams.has("seed")) {
+        //     p.set(KEY.SEED, tpParams.get("seed").asInt());
+        // }
+        // For some seeds, htm.java throws
+        // java.lang.ArrayIndexOutOfBoundsException: 1017
+        //    at org.numenta.nupic.Connections.computeActivity(Connections.java:1199)
```

Comment:
I think it would be better to open an Issue in HTM.Java, than making this comment a permanent feature inside this class. I don't think there's any reason to document bugs. They should just be fixed. IMHO